eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -127,7 +189,7 @@ jobs:
         env:
           TEST_MODE: "live"
           ${{ insert }}: ${{ parameters.EnvVars }}
-        condition: eq(variables['TestType'],'sample')
+        condition: and(eq(variables['TestType'],'sample'),eq(variables['DependencyVersion'],'repo'))
```

Comment:
Do we even set TestType to sample in any matrix entries? I couldn't find any in here.